var/spack/repos/builtin/packages/zee/package.py

```diff
@@ -27,6 +27,8 @@ class Zee(CMakePackage):
             description='Compile C++ with warnings')
     variant('petsc', default=True,
             description='Compile examples using PETSc')
+    variant('ci', default=False,
+            description='Indicate that build occurs during continuous integration')
     depends_on('cmake@3:', type='build')
     depends_on('pkg-config', type='build')
     depends_on('py-cmake-format', type='build', when='@develop')
```

Comment:
this and below three can become `when='@develop+ci'`?

Reply:
Sounds correct. Maybe not `+ci` but `+codechecks`? A little more expressive, but this will indicate that it's not just for Jenkins/Travis.

Reply:
I renamed `ci` variant to `codechecks` but IMO the `when` conditions are independent of the version and should only require `+codechecks`

Reply:
ok. those py-cmake-format and pre-commit packages are very small so I wouldn't mind.